### PR TITLE
Replace -activate tags on Anticipate/Forewarn with -ability

### PR DESCRIPTION
**Pull request by ascriptmaster**
Related: https://github.com/Zarel/Pokemon-Showdown-Client/pull/435

### CODE REVIEW

**Comment by Zarel** [comment]
See my comment on the client pullreq.

File: data/abilities.js [hunk 1]
```diff
@@ -878,7 +878,7 @@ exports.BattleAbilities = {
 			}
 			if (!warnMoves.length) return;
 			var warnMove = warnMoves[this.random(warnMoves.length)];
-			this.add('-activate', pokemon, 'ability: Forewarn', warnMove[0], warnMove[1]);
+			this.add('-activate', pokemon, 'Forewarn', warnMove[0], '[of] ' + warnMove[1]);
```
Comment (Zarel) [inline data/abilities.js]: This should be `ability: Forewarn`, not `Forewarn`. If you don't put any tag in front of an effect, it's a volatile status.

Comment (Zarel) [inline data/abilities.js]: Feel free to use `"[from] move: " + warnMove[0]` here.

**Comment by ascriptmaster** [comment]
I don't know why, but it seems odd to use `[from]` for Forewarn since it was activated by the switch-in and not by the move. (And I'd have to submit another client change.)

**Pull request merged**